shell/core/types.ts

```diff
@@ -693,6 +699,8 @@ export interface IPlugin {
   ): void;
   addNavHooks(hooks: NavHooks): void;
 
+  enableServerSidePagination(config: ServerSidePaginationExtensionConfig): void;
```

Comment:
I think we are missing "typing" `enableServerSidePagination` into `IPlugin` (as optional), if I am not mistaken @richard-cox . 

That's the error I was getting in the `ui-plugin-examples` repo

`Property 'enableServerSidePagination' does not exist on type 'IPlugin'`